ubtue_Tidsskrift.js

```diff
@@ -5,11 +5,11 @@
 	"target": "^https?://tidsskrift.dk/[^/]+/(article|issue)/view.*/[0-9]+",
 	"minVersion": "3.0",
 	"maxVersion": "",
-	"priority": 80,
-	"inRepository": false,
+	"priority": 150,
```

Comment:
Why 150 now ?

Reply:
This translator is not used any more because it has been replaced by ubtue_Open Journal Systems Standard, the priority is now 900, according to the example above.